internal/controllers/purchase_orders/purchase_orders_default.go

```diff
@@ -0,0 +1,35 @@
+package purchase_orders_controller
+
+import (
+	"github.com/maxwelbm/alkemy-g6/internal/models"
+)
+
+type PurchaseOrdersController struct {
+	Service models.PurchaseOrdersService
+}
```

Comment:
só por questão de padrão mesmo, que tal usar `sv`?
```suggestion
type PurchaseOrdersController struct {
	sv models.PurchaseOrdersService
}
```